gui/wxpython/lmgr/layertree.py

```diff
@@ -1742,7 +1742,7 @@ def OnDeleteLayer(self, event):
 
         try:
             item.properties.Close(True)
-        except:
+        except Exception:
```

Comment:
Try specific errors here

Reply:
I only see `AttributeError` being applicable here for the `Close` property